src/main/java/hudson/plugins/jira/model/JiraVersion.java

```diff
@@ -20,8 +21,9 @@ public JiraVersion(String name, Calendar releaseDate, boolean released, boolean
         this.archived = archived;
     }
 
-    public JiraVersion(String name, Calendar startDate, Calendar releaseDate, boolean released, boolean archived) {
+    public JiraVersion(String name, String description, Calendar startDate, Calendar releaseDate, boolean released, boolean archived) {
```

Comment:
please always leave the previous constructor and marl it as deprecated (always backward compat)